### ISSUE TITLE
Improve Mathematical Ops: SIN/EXP2/LOG2

### ISSUE THREAD

**Issue by hikettei** [hikettei/Caten#351]
- [ ] Benchmark: Softmax (vs PyTorch)
- [ ] SLEEF=1 for Clang
- [ ] Or TRANSCENDENTAL=1 to replace them w/ approximated mathematical ops
